Cargo.toml

```diff
@@ -22,23 +22,23 @@ name = "pikelet"
 required-features = ["cli"]
 
 [dependencies]
-codespan = "0.1.1"
+codespan = "0.1.2"
 codespan-reporting = "0.1.3"
 failure = "0.1.1"
-im = "10.2"
-lalrpop-util = "0.15.1"
-moniker = { git = "https://github.com/brendanzab/moniker", rev = "5fe9fae", version = "0.1.0" }
-pretty = { version = "0.5.1", features = ["termcolor"] }
+im = "11.0.0"
+lalrpop-util = "0.15.2"
+moniker = { git = "https://github.com/brendanzab/moniker", rev = "2fcdbf1", version = "0.1.0", features = ["codespan"] }
```

Comment:
Just testing this at the moment, we'll have to release a new version of Moniker.